resources/definitions/koonovo_pyramid.def.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "KOONOVO Pyramid",
+    "version": 2,
+    "inherits": "koonovo_base",
+    "overrides": {
+        "machine_name": { "default_value": "KOONOVO Pyramid" },
+        "machine_width": { "default_value": 310 },
+        "machine_depth": { "default_value": 310 },
+        "machine_height": { "default_value": 400 },
```

Comment:
Alibaba lists the size of this printer as 300x300x400: https://www.alibaba.com/product-detail/3d-Desktop-3D-Printer-Kits-KOONOVO_62404380206.html?spm=a2700.galleryofferlist.normal_offer.d_title.39704c94hBUbHe